### DISCUSSION TITLE
ElastAlert Status Error

### DISCUSSION THREAD

**Discussion by Hemro** [Security-Onion-Solutions/securityonion#3772] [Unsupported Versions] [answered]
When I use "so-start elastalert" it runs fine and changes state to running but when I check status it shows as [error]. There are no errors in /opt/so/log/elastalert log and cannot find anything relevant when running "/usr/sbin/so-elastic-diagnose"

![image](https://user-images.githubusercontent.com/22611477/113738436-cbc8fc80-96f6-11eb-80a0-a53d76a87e02.png)

![image](https://user-images.githubusercontent.com/22611477/113738175-97edd700-96f6-11eb-92d8-dce967d5791c.png)

**Comment by jertel** [comment, marked as answer]
Check the `docker logs so-elastalert` output for possible errors.

**Comment by Hemro** [comment]
It was a parsing block mapping error
